ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/a/AssetWizardSpec.groovy

```diff
@@ -33,8 +34,7 @@ class AssetWizardSpec extends AbstractModelCatalogueGebSpec {
     def "go to login"() {
         when:
         loginAdmin()
-        select "Test 1"
-        selectInTree "Test 1"
+        select 'Rare Disease Conditions'
```

Comment:
there is no *Rare Disease Conditions* change set in the functional test environment